docs/notebooks/api_user_guide/1_overview.ipynb

```diff
@@ -0,0 +1,4478 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Overview"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This overview demonstrates the basic capabilities of `eodag` in a simple worflow, whose steps are introduced in more details in the following pages.\n",
+    "\n",
+    "The workflow consists of the following steps:\n",
+    "\n",
+    "1. [Configure](#Configure): `eodag` is configured to use the provider *PEPS* (registering to *PEPS* is required to download the products, see [how to register to a provider](../../getting_started_guide/register.rst))\n",
+    "2. [Search](#Search): *Sentinel 2 Level-1C* products (i.e. images) are searched for over an area in France in March 2021.\n",
+    "3. [Crunch](#Crunch): A decicated filter - that we call *cruncher* - provided by `eodag`  is used to select products with a cloud cover less than 10%.\n",
+    "4. [Serialize](#Serialize): Save the filtered products as a GeoJSON file.\n",
+    "4. [Download](#Download): The products are downloaded.\n",
+    "5. [Post-process](#Post-process): `eodag-cube` is an external package that is used to access a product's data, it is going to be used to calculate the NDVI of a product."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configure"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The minimum required configuration to run this workflow is to set the credentials for *PEPS*. They will be used to download the products."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"EODAG__PEPS__AUTH__CREDENTIALS__USERNAME\"] = \"PLEASE_CHANGE_ME\"\n",
+    "os.environ[\"EODAG__PEPS__AUTH__CREDENTIALS__PASSWORD\"] = \"PLEASE_CHANGE_ME\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A workspace directory is created to store the downloaded products."
+   ]
```

Comment:
```suggestion
   "source": [
    "Now we will configure `eodag` to be able to download using *PEPS*. For that we need to fill our credentials:\n",
    "- in the user configuration file `~/.config/eodag.eodag.yml`:\n",
    "    ```yaml\n",
    "    peps:\n",
    "        priority: # Lower value means lower priority (Default: 1)\n",
    "        search:  # Search parameters configuration\n",
    "        download:\n",
    "            extract:  # whether to extract the downloaded products (true or false).\n",
    "            outputs_prefix: # where to store downloaded products.\n",
    "            dl_url_params:  # additional parameters to pass over to the download url as an url parameter\n",
    "        auth:\n",
    "            credentials:\n",
    "                username: PLEASE_CHANGE_ME\n",
    "                password: PLEASE_CHANGE_ME\n",
    "    ```\n",
    "- or using environment variables: (we also set `outputs_prefix`, the directory where to store downloaded products)"
   ]
```